File: despatch.WEB/app/src/swagger/models/user-organization-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { OrganizationResponse } from './organization-response';
import { UserResponse } from './user-response';
export interface UserOrganizationResponse extends BaseEntityResponse{
  contactDataId?: string;
  organization?: OrganizationResponse;
  organizationId?: string;
  user?: UserResponse;
}

File: despatch.Domain/Models/Default/Response/UserOrganizationResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class UserOrganizationResponse : BaseEntityResponse
    {
        public string ContactDataId { get; set; }
        public string OrganizationId { get; set; }

        public virtual UserResponse User { get; set; }
        public virtual OrganizationResponse Organization { get; set; }
    }
}
